bulgarian/burning_room.md

```diff
@@ -15,3 +15,5 @@
 6. [Грабваш пистолета и започваш да търсиш мафиотите съм.](./terminator/terminator.md)
 
 7. [Завиваш се с одеалото и се опитваш да набереш някой близък човек.](./under_the_covers/under_the_covers.md)
+
+8. [Бръкваш в джоба и изпиваш хапчето super man.](./superman/superman.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.